both/collections/round.js

```diff
@@ -30,6 +30,13 @@ _.extend(Round.prototype, {
     assert(this.eventCode);
     return wca.eventByCode[this.eventCode].solveTimeFields;
   },
+  roundCode: function() {
+    if(this.isLast()) {
+      return this.softCutoff ? 'c' : 'f';
+    } else {
+      return (this.softCutoff ? 'degc' : '123f')[this.nthRound-1];
```

Comment:
I don't have a strong sense of what should be in wca.js and why.

It vaguely makes sense to me as some kind of catalog of external dependencies. Parts of it encodes WCA regulations. Others reflect the WCA DB schema. But I don't sense the organizing principle.

More concretely, where would this code go? `wca.roundByCode` seems closest, but it's an awkward fit. You could also imagine all of  `wca.roundByCode` living in the Round class. But then its "wcaness" is lost.

Also, `wca.supportedRounds` can be deleted.